### DISCUSSION TITLE
Counting Objects in Zone IN Real Time

### DISCUSSION THREAD

**Discussion by StefanoGarzoni** [roboflow/notebooks#200] [Q&A] [answered]
Hi everyone, while I was waiting for your answer to my last question (for 3D position objects) I saw this video (https://www.youtube.com/watch?v=l_kf9CfZ_8M)
where you explain the use of a "zone counter" and I was wondering if it was possible and if so how can I use this tool in real time (with the video coming from my webcam) I await your response, as always I thank you I advance and look forward to your help!

**Comment by SkalskiP** [comment]
Hi, @Stefano0513! 👋🏻 I noticed that you already closed this discussion. I hope you found the answer you were looking for. But if not, here is a tutorial where I show exactly what you are looking for: https://youtu.be/QV85eYOb7gk?si=zxQamz0N8vBbIiq4

Best of luck!

**Reply by StefanoGarzoni** [reply, marked as answer]
Hi, I closed the discussion because looking back on it it's a really silly question, I simply took my old code with the line counter and made some changes by inserting the zone object in its place... it was more my mistake of haste and insecurity to ask for help, in any case, thanks again for all the help you have given me and are giving me!

if it's not too much to ask, I would be really interested in understanding how to detect the 3D position in the space of objects, I had already opened a discussion about it, if you have time I would gladly appreciate it and I would need a great help. Thank you again!

**Reply by SkalskiP** [reply]
Could you send me a link to that other discussion?

**Reply by StefanoGarzoni** [reply]
here : 
https://github.com/roboflow/notebooks/discussions/199
